src/redux/categories/catagoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under Constructions';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => initialState,
+  },
+});
+
```

Comment:
Your initialState needs to have a state value categories that contains an empty array, something like this: 
```javascript
  initialState: {
    categories: [],
  },
```
The same implies for the booksSlice.js
Kindly fix these issue 👍